### PR TITLE
EX-1752 Add seatbid array support

### PR DESCRIPTION
**Pull request by mikhalovich**

### CODE REVIEW

File: test/spec/modules/sovrnBidAdapter_spec.js [hunk 1]
```diff
@@ -307,206 +277,251 @@ describe('sovrnBidAdapter', function() {
...
+
+        expect(impression.ext.data.pbadslot).to.equal('homepage-top-rect')
+        expect(impression.ext.data.adUnitSpecificAttribute).to.equal('123')
+        expect(impression.ext.deals).to.deep.equal(['seg1', 'seg2'])
       })
     })
-  });
+  })
 
   describe('interpretResponse', function () {
```
Comment (jrosendahl) [inline test/spec/modules/sovrnBidAdapter_spec.js]: I am not seeing the unittest for above code.

**Review by jrosendahl — CHANGES_REQUESTED** [review]

File: test/spec/modules/sovrnBidAdapter_spec.js (hunk 1 above)
Comment (mikhalovich) [reply test/spec/modules/sovrnBidAdapter_spec.js]: I added unit test for new functionality [here](https://github.com/sovrn/Prebid.js/pull/34/commits/4e4d192cd7004ce00841dc7a8750eb9edbd9c1f9#diff-9b3cbd20715f7d1fc68496aca92c279070d2a0bc1d5cc004c47792272d1e39adR436)

**Review by gannonk08 — APPROVED** [review]

File: test/spec/modules/sovrnBidAdapter_spec.js (hunk 1 above)
Comment (mikhalovich) [reply test/spec/modules/sovrnBidAdapter_spec.js]: I have updated files. Now tests [here](https://github.com/sovrn/Prebid.js/pull/34/files#diff-9b3cbd20715f7d1fc68496aca92c279070d2a0bc1d5cc004c47792272d1e39adR474)

**Review by jrosendahl — COMMENTED** [review]
We need to get an integration test framework together before this can move forward

**Comment by maximpakhomov** [comment]
@cpabst This PR into the Prebid.js repository was merged yesterday. What should be done with this PR next?

**Comment by cpabst** [comment]
@maximpakhomov 
@jrosendahl is looking at a couple of things here. Once he completed that review, will probably close this PR and update from prebid.

**Comment by cpabst** [comment]
Merged into prebid, so closing this PR and updating.

**Pull request closed without merging**